rasa/core/policies/rule_policy.py

```diff
@@ -202,12 +190,35 @@ def predict_action_probabilities(
             )
 
         if possible_keys:
+            # TODO rethink that
             key = max(possible_keys, key=len)
 
             recalled = self.lookup.get(key)
+
+            if active_form_name:
+                predicted_listen_from_general_rule = recalled is None or (
+                    domain.action_names[recalled] == ACTION_LISTEN_NAME
+                    and f"active_form_{active_form_name}" not in key
+                )
+                if predicted_listen_from_general_rule:
+                    logger.debug(f"Predicted '{active_form_name}'")
+                    result[domain.index_for_action(active_form_name)] = 1
+                    return result
+
+                predicted_form_from_form_rule = (
```

Comment:
let's add a comment here that this is a rule for an unhappy form path and also why we need to disable `FormValidation`

Reply:
sure. so the name `...form_from_form...` is good 😂